docs/operator.md

```diff
@@ -236,6 +236,16 @@ in [cross-site replication](replication.md#operator-replication) |
 | **Example**     | `rack: rack-22` |
 | **Description** | [Labels are key-value pairs attached to objects](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) |
 |                 | |
+| **Key**         | {{ optionlink('pxc.serviceLabels') }} |
```

Comment:
I don't think we should merge this, this was removed and pxc.expose.labels and pxc.expose.annotations should be used. This is probably some misunderstanding.
I think we should document that these options are deprecated and will be removed after some version.